site/class-1-load/.metadata.json

```diff
@@ -1,5 +0,0 @@
-{
```

Comment:
Dude! You’re removing our website!

Could you add this file back? You can do that by clicking on the pencil in the top-right corner here.

Reply:
Tip: either paste this file back (the one that’s currently here). Or throw this branch out and try again.